framework/components/ecl-breadcrumbs/_breadcrumbs.scss

```diff
@@ -179,46 +179,46 @@ a.breadcrumb__link {
     }
   }
 
-  // Indent the last segment, leaving space for .breadcrumb__btn-separator
-  .breadcrumb__segment--last {
+  // Indent the last segment, leaving space for .ecl-breadcrumb__btn-separator
+  .ecl-breadcrumb__segment--last {
     padding-left: 60px;
   }
 
-  // In case of 2 .breadcrumb__segment-s, don't indent the second one.
-  .breadcrumb__segment:first-child:nth-last-child(2),
-  .breadcrumb__segment:first-child:nth-last-child(2) ~ .breadcrumb__segment {
+  // In case of 2 .ecl-breadcrumb__segment-s, don't indent the second one.
+  .ecl-breadcrumb__segment:first-child:nth-last-child(2),
+  .ecl-breadcrumb__segment:first-child:nth-last-child(2) ~ .ecl-breadcrumb__segment {
     padding-left: 0;
   }
 
   // List of breadcrumbs is open.
   &.is-open {
-    .breadcrumb__link {
+    .ecl-breadcrumb__link {
       display: block;
     }
 
-    .breadcrumb__text {
+    .ecl-breadcrumb__text {
       max-width: none;
       white-space: normal;
       overflow: visible;
     }
 
-    .breadcrumb__segment {
+    .ecl-breadcrumb__segment {
       display: block;
     }
 
-    .breadcrumb__segment--last {
+    .ecl-breadcrumb__segment--last {
       padding-left: 0;
       transition: all 0.3s ease 0s;
     }
   }
 
   @include ecl-media-breakpoint-up(xs) {
-    .breadcrumb__segment {
+    .ecl-breadcrumb__segment {
       &.is-hidden {
         display: none;
       }
 
-      &.breadcrumb__segment--next-hidden {
+      &.ecl-breadcrumb__segment--next-hidden {
         padding-right: 25px;
```

Comment:
I didn't know which one to use out of https://github.com/ec-europa/europa-component-library/blob/master/framework/base/variables/_spacing.scss honestly without having a hint which one is close.